contracts/loot/item/constants.cairo

```diff
@@ -6,54 +6,257 @@
 
+namespace Material:
+    const generic = 0
+
+    namespace Metal:
+        const generic = 0
+        const ancient = 1
+        const holy = 2
+        const ornate = 3
+        const gold = 4
+        const silver = 5
+        const bronze = 6
+        const platinum = 7
+        const titanium = 8
+        const steel = 9
+    end
+    namespace Cloth:
+        const generic = 0
+        const royal = 1
+        const divine = 2
+        const brightsilk = 3 
+        const silk = 4
+        const wool = 5
+        const linen = 6
+    end
+    namespace Biotic:
+        namespace Demon:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Dragon:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const skin = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Animal:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Human:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+    end
+    namespace Paper:
+        const generic = 0
+        const magical = 1
+    end
+    namespace Wood:
+        namespace Hard:
+            const generic = 0
+            const walnut = 1
+            const mahogany = 2
+            const maple = 3
+            const oak = 4
+            const rosewood = 5
+            const cherry = 6
+            const balsa = 7
+            const birch = 8
+        end
+        
+        namespace Soft:
+            const generic = 0
+            const cedar = 1
+            const pine = 2
+            const fir = 3
+            const hemlock = 4
+            const spruace = 5
+        end
+    end
+end
+
+# Weight modifiers expressed as percentage of base material
+# LH TODO: Can't use non-ints within namespace so need to figure out proper way to structure the data below
+namespace WeightModifier:
```

Comment:
want to store this as 1.5 so it can be more easily used in material density calculation